snapshots/one/2024-10-02/oda_data_extract.py

```diff
@@ -0,0 +1,294 @@
+"""
```

Comment:
Hi @Marigold! I came back to this script and can't make it run in the ETL. It doesn't find oda_data even if I run `pip install oda_data --upgrade`. In the staging server it finds an incompatibility with `frictionless`:
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
etl 0.1.0 requires frictionless[pandas]<5.0.0,>=4.40.8, but you have frictionless 5.18.0 which is incompatible.
Successfully installed bblocks-1.4.0 camelot-py-0.11.0 country-converter-1.3 frictionless-5.18.0 hdx-python-country-3.8.5 hdx-python-utilities-3.7.4 humanize-4.11.0 ijson-3.3.0 imf-reader-1.1.0 jsonlines-4.0.0 jsonpath_ng-1.7.0 libhxl-5.2.2 loguru-0.7.2 oda-reader-1.0.0 oda_data-1.5.0 opencv-python-4.10.0.84 ply-3.11 pyarrow-16.1.0 pydeflate-2.1.0 pyjstat-2.4.0 pypdf-3.17.4 pyphonetics-0.5.3 python-io-wrapper-0.3.1 rapidfuzz-3.10.1 ratelimit-2.2.1 requests-file-2.1.0 tableschema-to-template-0.0.13 thefuzz-0.22.1 webdriver-manager-4.0.2 wheel-0.45.1 xlrd3-1.1.0 xlsx2csv-0.8.4 xlsxwriter-3.2.0 xlwt-1.3.0
```

Reply:
@paarriagadap I've upgraded frictionless, could you rebase / merge and try again, please?

Reply:
I was able to run it on a staging server with
```
ssh owid@owid@staging-site-oda-one-data-updated
cd etl
uv run pip install oda_data --upgrade
uv run python snapshots/one/2024-10-02/oda_data_extract.py
```
aren't you missing `uv run` when running it locally?

Reply:
Yes, I can run it on staging.

Locally, for me running with and without `uv run` is the same thing for me. Should it not be that case? I automatically activate the environment and I can always do that, also when we had poetry

Reply:
Yes, it should be the same, I just thought that the error could be related to using different python environment.

Is it enough if you can run it on staging for now?

Reply:
Hi, it's ok, I can run it on staging. Thanks!